parking_permits/views.py

```diff
@@ -806,6 +817,13 @@ def delete(self, request, *args, **kwargs):
                 transaction.set_rollback(True)
         return Response(status=204)
 
+    def is_valid_uuid(self, value):
+        try:
+            uuid.UUID(str(value))
+            return True
+        except ValueError:
```

Comment:
You should also check for AttributeError when making a UUID field, see https://github.com/django/django/blob/main/django/db/models/fields/__init__.py#L2746

Reply:
Updated this.